tests-e2e/cypress/integration/adminconsole/analytics_spec.js

```diff
@@ -0,0 +1,101 @@
+            });
+            cy.apiCreatePlaybook({
+                teamId: testTeam.id,
+                title: 'Playbook',
+                memberIDs: [],
+            }).then((playbook) => {
+                testPlaybook = playbook;
+            });
+        });
+    });
+
+    describe('site statistics', () => {
+        it('playbooks and runs counters are visible', () => {
+            cy.apiLogin(testSysadmin);
+
+            // # Go to admin console > site statistics
+            cy.visit('/admin_console/reporting/system_analytics');
+            cy.findByTestId('playbooks.playbook_count').should('exist');
+            cy.findByTestId('playbooks.playbook_run_count').should('exist');
+        });
+
+        it('playbook counter increases after creating a playbook', () => {
+            cy.apiLogin(testSysadmin);
+            let counter;
+
+            // # Go to admin console > site statistics
+            cy.visit('/admin_console/reporting/system_analytics');
+
+            // # Capture current value of playbook counter
+            cy.findByTestId('playbooks.playbook_count').invoke('prop', 'innerText').then((pbCount) => {
+                counter = parseInt(pbCount, 10);
+                cy.apiLogin(testUser);
+
+                // # Create a playbook
+                cy.apiCreatePlaybook({
+                    teamId: testTeam.id,
+                    title: 'Playbook',
+                    memberIDs: [],
+                }).then(() => {
+                    cy.apiLogin(testSysadmin);
+
+                    // # Go to admin console > site statistics
+                    cy.visit('/admin_console/reporting/system_analytics');
+
+                    // * Playbook Counter has been increased by 1
+                    cy.findByTestId('playbooks.playbook_count').contains(String(counter + 1));
+                });
+            });
+        });
+
+        it('run counter increases after creating a playbook', () => {
+            cy.apiLogin(testSysadmin);
+            let counter;
+
+            // # Go to admin console > site statistics
+            cy.visit('/admin_console/reporting/system_analytics');
+
+            // # Capture current value of run counter
+            cy.findByTestId('playbooks.playbook_run_count').invoke('prop', 'innerText').then((runCount) => {
+                counter = parseInt(runCount, 10);
+                cy.apiLogin(testUser);
+
+                // # create a run
+                cy.apiRunPlaybook({
+                    teamId: testTeam.id,
+                    playbookId: testPlaybook.id,
+                    playbookRunName: 'My run for test',
+                    ownerUserId: testUser.id,
+                }).then(() => {
+                    cy.apiLogin(testSysadmin);
+
+                    // # Go to admin console > site statistics
+                    cy.visit('/admin_console/reporting/system_analytics');
+
+                    // * Run Counter has been increased by 1
```

Comment:
Same super nit here:
```suggestion
                    // * Verify that the Run Counter has been increased by 1
```